src/lib/components/Channel/Stream/StreamControls.svelte

```diff
@@ -399,22 +405,31 @@
 		</button>
 
 		{#if $is_feature_restream_enabled}
-			<button
-				class="btn text-white border-none tooltip font-normal normal-case {$is_sharing_obs
-					? 'btn-primary'
-					: 'btn-neutral'}"
-				data-tip="Restream"
-				on:click={() => {
-					$is_sharing_obs = !$is_sharing_obs
-				}}
-				disabled={$is_sharing_screen ||
-					$is_sharing_webcam ||
-					$is_sharing_audio ||
-					!isHostOrGuest ||
-					!isChannelSocketConnected ||
-					!videoItemIsActive}>
-				<IconRestream />
-			</button>
+			<div class="drawer drawer-end">
```

Comment:
The drawer should be in its own file similar to the other drawers. DrawerEditChannel, DrawerChat, etc.